### PR TITLE
Add Serverless on HyperShift

### PR DESCRIPTION
**Pull request by liswang89**
Jira ticket:

https://issues.redhat.com/browse/LPTOCPCI-320

### CODE REVIEW

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-rhpit-interop-tests-main-serverless-hypershift-4.13-lp-rosa-hypershift-serverless-aws-rosa-hypershift

File: ci-operator/config/rhpit/interop-tests/rhpit-interop-tests-main__serverless-hypershift-4.13-lp-rosa-hypershift.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  '*':
+    limits:
+      memory: 8Gi
+    requests:
+      cpu: "1"
+      memory: 4Gi
+tests:
+- as: serverless-aws-rosa-hypershift
+  cron: 0 10 * * 1
+  remote_api: true
```
Comment (ascerra) [inline ci-operator/config/rhpit/interop-tests/rhpit-interop-tests-main__serverless-hypershift-4.13-lp-rosa-hypershift.yaml:27]: This line can be removed

File: ci-operator/config/rhpit/interop-tests/rhpit-interop-tests-main__serverless-hypershift-4.13-lp-rosa-hypershift.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      FIREWATCH_CONFIG: |
+        [
+          {"step": "aws-*", "failure_type": "all", "classification": "Infrastructure AWS", "jira_project": "LPTOCPCI"},
+          {"step": "gather-*", "failure_type": "all", "classification": "Other", "jira_project": "LPTOCPCI"},
+          {"step": "osd-*", "failure_type": "all", "classification": "User Setup", "jira_project": "LPTOCPCI"},
+          {"step": "rosa-*", "failure_type": "all", "classification":  "Cluster Creation", "jira_project": "LPTOCPCI"}
```
Comment (ascerra) [inline ci-operator/config/rhpit/interop-tests/rhpit-interop-tests-main__serverless-hypershift-4.13-lp-rosa-hypershift.yaml:36]: You can add two more lines to the firewatch config to cover the install-operators and cucushift-hypershift health check steps

```          {"step": "install-operators", "failure_type": "all", "classification": "Infrastructure Provisioning - Operators", "jira_project": "LPTOCPCI"},
          {"step": "cucushift-hypershift-*", "failure_type": "all", "classification": "Cluster Health Check", "jira_project": "LPTOCPCI"}
```

File: ci-operator/config/rhpit/interop-tests/rhpit-interop-tests-main__serverless-hypershift-4.13-lp-rosa-hypershift.yaml (hunk 1 above)
Comment (liswang89) [reply ci-operator/config/rhpit/interop-tests/rhpit-interop-tests-main__serverless-hypershift-4.13-lp-rosa-hypershift.yaml]: Removed.

File: ci-operator/config/rhpit/interop-tests/rhpit-interop-tests-main__serverless-hypershift-4.13-lp-rosa-hypershift.yaml (hunk 2 above)
Comment (liswang89) [reply ci-operator/config/rhpit/interop-tests/rhpit-interop-tests-main__serverless-hypershift-4.13-lp-rosa-hypershift.yaml]: Updated as suggested.

**Pull request merged**